qa/rpc-tests/paymentdisclosure.py

```diff
@@ -74,7 +74,7 @@ def run_test (self):
         txid = wait_and_assert_operationid_status(self.nodes[0], myopid)
 
         # Check the tx has joinsplits
-        assert( len(self.nodes[0].getrawtransaction("" + txid, 1)["vjoinsplit"]) > 0 )
+        assert( len(self.nodes[0].getrawtransaction("" + txid, 1)["vJoinSplit"]) > 0 )
```

Comment:
Hmm, I hadn't realized this was used in `getrawtransaction` output. Changing that is strictly speaking a compatibility break. I think the original intent of #3188 was just to change it in the C++ code.

Reply:
Reverted this instance. Let me know if there are any other compatibility breaks.